src/local/butler/reproduce_tool/android.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Android emulator installation and management."""
+from __future__ import print_function
+
+import os
+import time
+
+from local.butler.reproduce_tool import errors
+from local.butler.reproduce_tool import prompts
+from platforms.android import adb
+from platforms.android import device
+from system import environment
+from system import new_process
+
+ADB_DEVICES_SEPARATOR_STRING = 'List of devices attached'
+EMULATOR_PATH = os.path.join(
+    os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, os.pardir,
```

Comment:
this looks a bit messy. how about using ROOT_DIR ?

Reply:
+1

Reply:
Done